tests/integration/suite/sentry/metrics/expiry.go

```diff
@@ -49,8 +50,14 @@ type expiry struct {
 func (e *expiry) Setup(t *testing.T) []framework.Option {
 	rootKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
 	require.NoError(t, err)
+	jwtKey, err := rsa.GenerateKey(rand.Reader, 2048)
+	require.NoError(t, err)
+
 	onemonth := time.Hour * 24 * 30
-	bundle, err := ca.GenerateBundle(rootKey, "integration.test.dapr.io", time.Second*5, &onemonth)
+	bundle, err := ca.GenerateBundle(rootKey, jwtKey, "integration.test.dapr.io", time.Second*5, &onemonth, ca.CredentialGenOptions{
+		RequireX509: true,
+		RequireJWT:  true,
+	})
```

Comment:
Are there any metrics relating to JWT signing we should be testing?

Reply:
I haven't added any. Can do, if they are required